README.md

```diff
@@ -1,14 +1,165 @@
-# UserManagementService
+# User Management Service
```

Comment:
Root level README.md should contain endpoint details, link to API spec, collection, coverage report, etc.
Please update to include required information on the service and not how to develop the service.
Also make sure to highlight any internal dependencies, communicating services(in this to activate the user, etc.).

Reply:
done

Reply:
Still not reflected at Root level README.